nautobot_secrets_providers/providers/__init__.py

```diff
@@ -1,12 +1,14 @@
 """Nautobot Secrets Providers."""
 
 from .aws import AWSSecretsManagerSecretsProvider
+from .aws_ssm import AWSSystemsManagerParameterStore
 from .hashicorp import HashiCorpVaultSecretsProvider
 from .delinea import ThycoticSecretServerSecretsProviderId, ThycoticSecretServerSecretsProviderPath
 
 __all__ = (  # type: ignore
     AWSSecretsManagerSecretsProvider,  # pylint: disable=invalid-all-object
     HashiCorpVaultSecretsProvider,  # pylint: disable=invalid-all-object
+    AWSSystemsManagerParameterStore,  # pylint: disable=invalid-all-object
     ThycoticSecretServerSecretsProviderId,  # pylint: disable=invalid-all-object
     ThycoticSecretServerSecretsProviderPath,  # pylint: disable=invalid-all-object
```

Comment:
These could just be strings and then there's no need for the `pylint: disable` comments.

Reply:
Nope, because of https://github.com/nautobot/nautobot-plugin-secrets-providers/blob/develop/nautobot_secrets_providers/secrets.py#L15

Refactoring that is out of scope of this PR.